src/go/k8s/apis/redpanda/v1alpha1/redpanda_clusterspec_types.go

```diff
@@ -99,7 +99,7 @@ type RedpandaConsole struct {
 	Deployment *ConsoleCreateObj `json:"deployment,omitempty"`
 
 	// +kubebuilder:pruning:PreserveUnknownFields
-	Config *runtime.RawExtension `json:"config,omitempty"`
+	Console *runtime.RawExtension `json:"console,omitempty"`
```

Comment:
Why are we removing config?

Reply:
OK unblocking since I figured out that we are trying to really replicate what is found in the console subchart which is
```
console:
  config {}
 ```
 
 Before we merge this PR then, let's make the adjustment to the values file of the helm chart. Once that is complete Ill Merge that PR.